### PR TITLE
Complete exercise 5

### PR DESCRIPTION
**Pull request by jacobschweitzer**

### CODE REVIEW

File: src/exercise/05.js [hunk 1]
```diff
@@ -4,24 +4,16 @@
...
-// 🐨 add a className prop to each of these and apply the correct class names
-// 💰 Here are the available class names: box, box--large, box--medium, box--small
-
-// 🐨 add a style prop to each of them as well so their background color
-// matches what the text says it should be as well as `fontStyle: 'italic'`
-const smallBox = <div>small lightblue box</div>
-const mediumBox = <div>medium pink box</div>
-const largeBox = <div>large orange box</div>
+function Box(props) {
+  return <div className={"box box--" + props.boxSize} style={{backgroundColor: props.boxColor}}>{props.boxText}</div>
```
Comment (thischrisblack) [inline src/exercise/05.js]: Curious why concatenate and not `string template ${literal}`?

Comment (thischrisblack) [inline src/exercise/05.js]: Destructuring `{ boxSize, BoxColor } = props` would make the markup a bit more manageable?

File: src/exercise/05.js [hunk 2]
```diff
@@ -4,24 +4,16 @@
...
 
 function App() {
   return (
     <div>
-      {smallBox}
-      {mediumBox}
-      {largeBox}
+      <Box boxSize="small" boxColor="lightblue" boxText="small lightblue box"></Box>
+      <Box boxSize="medium" boxColor="pink" boxText="medium pink box"></Box>
+      <Box boxSize="large" boxColor="orange" boxText="large orange box"></Box>
```
Comment (thischrisblack) [inline src/exercise/05.js]: I could be wrong but I don't think this `boxText` approach is a normal pattern for React. If you do 

```
<Box>
  Hey!
</Box>
```

Then `'Hey!` gets passed in as the `children` prop of `props`, like this:

image

If you do

```
<Box>
  <span>Hey!</span>
  <span>Ho!</span>
</ Box>
```
Then the `children` prop of `props` will be an array with those `span` nodes, like so:

image

Again, I'm not sure if your solution is, like, anti-react, but I don't think I've seen it that way.

Comment (thischrisblack) [reply src/exercise/05.js]: Also, it doesn't meet the AC 😆 

image

**Review by thischrisblack — CHANGES_REQUESTED** [review]

File: src/exercise/05.js (hunk 1 above)
Comment (jacobschweitzer) [reply src/exercise/05.js]: Yes for sure, not sure why I didn't do it that way, old habits I suppose.

Comment (jacobschweitzer) [reply src/exercise/05.js]: No good reason, I'll make it a string literal, its easier to read.

**Pull request merged**